services/NotificationAgent/src/notification_agent/core.clj

```diff
@@ -4,105 +4,119 @@
         [clojure-commons.lcase-params :only [wrap-lcase-params]]
         [clojure-commons.query-params :only [wrap-query-params]]
         [service-logging.middleware :only [wrap-logging add-user-to-context]]
+        [compojure.api.middleware :only [wrap-exceptions]]
         [compojure.core]
         [korma.db :only [transaction]]
         [ring.middleware keyword-params nested-params]
         [notification-agent.delete]
         [notification-agent.notifications]
         [notification-agent.query]
+        [notification-agent.common :only [success-resp]]
         [notification-agent.seen]
-        [slingshot.slingshot :only [try+]])
+        [slingshot.slingshot :only [try+]]
+        [ring.util.http-response :only [ok]])
   (:require [compojure.route :as route]
             [clojure.tools.logging :as log]
-            [clojure-commons.error-codes :as ce]
+            [clojure-commons.exception :as cx]
             [notification-agent.config :as config]
             [notification-agent.db :as db]
             [ring.adapter.jetty :as jetty]
             [common-cli.core :as ccli]
             [me.raynes.fs :as fs]
-            [service-logging.thread-context :as tc]))
+            [service-logging.thread-context :as tc]
+            [cheshire.core :as cheshire]))
 
-(defn- trap
-  [ctx f]
-  (transaction (ce/trap ctx f)))
+(defn- success-resp
+  "Returns an empty success response."
+  ([]
+   (success-resp {}))
```

Comment:
Actually, the intent is different here, because a `(success-response nil)` should return an empty response body, but a `(success-response {})` should return a `{"success": true}` JSON response.